_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +784,1640 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
+  * *time_in_pool* - unsigned int (in binary form)
+* *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean; States if the result is obtained using the bootstrap mode, and is therefore not trusted (`true`), or when the daemon is fully synced (`false`).
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_txpool_backlog"}' -H 'Content-Type: application/json'
+
+{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "backlog": "...Binary...",
+    "status": "OK",
+    "untrusted": false
+  }
+}
+```
+
+
+### **get_output_distribution**
+
+
+
+Alias: *None*.
+
+Inputs:
+
+* *amounts* - array of unsigned int; amounts to look for
+* *cumulative* - boolean; (optional, default is `false`) States if the result should be cumulative (`true`) or not (`false`)
+* *from_height* - unsigned int; (optional, default is 0) starting height to check from
+* *to_height* - unsigned int; (optional, default is 0) ending height to check up to
+
+Outputs:
+
+* *distributions* - array of structure distribution as follows:
+  * *amount* - unsigned int
+  * *base* - unsigned int
+  * *distribution* - array of unsigned int
+  * *start_height* - unsigned int
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_output_distribution","params":{"amounts":[628780000],"from_height":1462078}}' -H 'Content-Type: application/json'
+
+{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "distributions": [{
+      "amount": 2628780000,
+      "base": 0,
+      "distribution": "",
+      "start_height": 1462078
+    }],
+    "status": "OK"
+  }
+}
+```
+
+
+---
+
+## Other Daemon RPC Calls
+
+Not all daemon RPC calls use the JSON_RPC interface. This section gives examples of these calls.
+
+The data structure for these calls is different than the JSON RPC calls. Whereas the JSON RPC methods were called using the `/json_rpc` extension and specifying a method, these methods are called at their own extensions. For example:
+
+    IP=127.0.0.1
+    PORT=18081
+    METHOD='gettransactions'
+	PARAMS='{"txs_hashes":["d6e48158472848e6687173a91ae6eebfa3e1d778e65252ee99d7515d63090408"]}'
+	curl \
+		-X POST http://$IP:$PORT/$METHOD \
+		-d $PARAMS \
+		-H 'Content-Type: application/json'
+
+Note: It is recommended to use JSON RPC where such alternatives exist, rather than the following methods. For example, the recommended way to get a node's height is via the JSON RPC methods [get_info](#getinfo) or [get_last_block_header](#get_last_block_header), rather than [getheight](#getheight) below.
+
+Some of them are ending with **.bin**: their data exchanged is in the form of binary, serialized objects, as defined in the [Core RPC Server](https://github.com/monero-project/monero/blob/master/src/rpc/core_rpc_server_commands_defs.h).
```

Comment:
Suggest: "For calls that end with **.bin**, the data is exchanged in the form of"...

Reply:
ok. I'm not a native english speaker, so i do trust you, and found it more elegant too.